Hello/view/frontend/templates/content.phtml

```diff
@@ -0,0 +1,43 @@
+<?php $collectionData = $block->getTestModel(); 
```

Comment:
"testmodel" use proper variable and function names
why is the word "test" used?